presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -101,19 +108,22 @@ Date and Time Functions
 
     Returns a timestamp with time zone from ``timestamp`` and ``zone``.
 
-.. function:: from_unixtime(unixtime) -> timestamp
+.. function:: from_unixtime(unixtime) -> timestamp(3)
 
-    Returns the UNIX timestamp ``unixtime`` as a timestamp. ``unixtime`` is the number of seconds since ``1970-01-01 00:00:00``.
+    Returns the UNIX timestamp ``unixtime`` as a ``timestamp(3)``. ``unixtime``
```

Comment:
Leave this part alone. It's fine to say "as a timestamp" since the function definition gives the exact type.

Reply:
@martint wanted to make sure we are explicit .. so I added it as he requested. Personally I think this helps. Can you guys decide together please..

Reply:
@martint specifically requested this change on all outputs ..